Automattic/pocket-casts-ios
# Playback 2023: Epilogue

| 📘 Part of: #1142 | 🎨 Designs: [Figma](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1599-20385&mode=design) |
|:---:|:---:|

Adds the 2023 visual for the Epilogue Story.

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/826d55c6-bc54-4174-bab2-e9eefeb1dd30" width="300">

## To test

1. Make sure you're logged in to an account that has a few episodes listened
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Go back to Profile
4. Tap the EOY image
5. Skip to the last story
5. ✅ Check that the story looks good
6. Tap "Play again"
7. ✅ You should go back to the first story
8. Give Patreon or Plus to your account and go to the last story again
9. ✅ The background should be slightly gold-ish

Please test on a big device (iPhone 15 Pro Max, for example), a normal device (iPhone 15), and a smaller device (iPod touch).

## Checklist
